composer.json

```diff
@@ -43,9 +43,9 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "0.5-dev"
+            "dev-master": "1.1-dev"
```

Comment:
I would use `1.0-dev` here. The reason is that commits after the first stable release will probably be bug fixes. So the following release will rather be `1.0.1` and not `1.1.0`. The branch alias can then be updated when the first new feature was merged (which requires to bump the minor version).